contracts/upgradeable_contracts/BasicHomeBridge.sol

```diff
@@ -144,4 +148,52 @@ contract BasicHomeBridge is EternalStorage, Validatable {
     function requiredMessageLength() public pure returns(uint256) {
         return Message.requiredMessageLength();
     }
+
+    function affirmationWithinLimits(uint256) internal view returns(bool) {
+        return true;
+    }
+
+    function onFailedAffirmation(address, uint256, bytes32) internal {
+    }
+
+    function outOfLimitAmount() public view returns(uint256) {
+        return uintStorage[keccak256(abi.encodePacked("outOfLimitAmount"))];
+    }
+
+    function setOutOfLimitAmount(uint256 _value) internal {
+        uintStorage[keccak256(abi.encodePacked("outOfLimitAmount"))] = _value;
+    }
+
+    function fixAssetsAboveLimits(bytes32 txHash, bool unlock_on_foreign) external onlyOwner {
```

Comment:
That makes sense. So when #111 is implemented, I'll update this method. It's that OK?

Reply:
The fix is quite easy:
```solidity
function fixAssetsAboveLimits(bytes32 txHash, bool unlock_on_foreign) external onlyProxyOwner {
```
Do you see any risks to implement it now?

Reply:
Thanks for pointing this out, I got confused and thought it would be a more complex solution. Here I updated the method https://github.com/poanetwork/poa-bridge-contracts/pull/112/commits/e4a3cf4d9d95a7f53d8aae7ca65dd749a934dc16